packages/expo/src/Google/Google.ts

```diff
@@ -20,33 +28,18 @@ type LogInResult =
       accessToken?: string;
       idToken: string | null;
       refreshToken: string | null;
-      user: {
-        id?: string;
-        name?: string;
-        givenName?: string;
-        familyName?: string;
-        photoUrl?: string;
-        email?: string;
-      };
+      user: GoogleUser;
     };
 
 export async function logInAsync(config: LogInConfig): Promise<LogInResult> {
   if (!AppAuth.authAsync) {
     throw new UnavailabilityError('AppAuth', 'logInAsync');
   }
 
-  const { behavior = 'web' } = config;
-
-  if (behavior !== 'web') {
-    if (Constants.appOwnership === 'expo') {
-      console.warn(
-        'Native Google Sign-In is only available in ExpoKit projects. Falling back to `web` behavior'
-      );
-    } else {
-      console.warn(
-        "Deprecated: Native Google Sign-In has been moved to Expo.GoogleSignIn ('expo-google-sign-in') Falling back to `web` behavior"
-      );
-    }
+  if (config.behavior !== undefined) {
+    console.warn(
+      "Deprecated: Native Google Sign-In has been moved to Expo.GoogleSignIn ('expo-google-sign-in') Falling back to `web` behavior"
```

Comment:
Either include a note saying when this will be removed (SDK 34) or a comment with the date this was deprecated so that when we look over this code in the future we can more easily see whether it can be removed.